lib/team.rb

```diff
@@ -0,0 +1,28 @@
+require 'csv'
+
+class Team
+
+  @@all = nil
+
+  def self.all
+    @@all
+  end
+
+  def self.from_csv(file_path)
+    csv = CSV.read("#{file_path}", headers: true, header_converters: :symbol)
+    @@all = csv.map do |row|
+      Team.new(row)
+    end
+  end
+
+  attr_reader :team_id, :franchise_id, :team_name, :abbreviation, :stadium, :link
+
+  def initialize(team_info)
+    @team_id = team_info[:team_id].to_i
+    @franchise_id = team_info[:franchiseid].to_i
+    @team_name = team_info[:teamname]
+    @abbreviation = team_info[:abbreviation]
+    @stadium = team_info[:stadium]
+    @link = team_info[:link]
+  end
```

Comment:
followed ruby convention for the variable naming

Reply:
we all need to be careful when calling headers like :franchiseid instead of :franchise_id

Reply:
@perryr16 I'm not sure how to be careful. Do you mean changing the instance variable to follow the correct convention?